plugins/parsers/csv/parser.go

```diff
@@ -197,6 +198,14 @@ outer:
 				value = strings.Trim(value, " ")
 			}
 
+			// don't record fields where the value matches a skip value
+			for _, s := range p.SkipValues {
+				fmt.Println("in here")
```

Comment:
Remove debugging println

Reply:
Done, I'll make sure I check more carefully for println in the future before I commit!

Reply:
No problem, they sneak in from time to time for everybody.  That's what reviews are for.